satpy/tests/reader_tests/test_viirs_l2.py

```diff
@@ -0,0 +1,133 @@
+        """Fill file contents with default data."""
+        if file_type.startswith("CLD"):
+            file_content["geolocation_data/latitude"] = DEFAULT_LAT_DATA
+            file_content["geolocation_data/longitude"] = DEFAULT_LON_DATA
+            if file_type == "CLDPRO":
+                file_content["geophysical_data/Cloud_Top_Height"] = DEFAULT_FILE_DATA
+            elif file_type == "CLDMSK":
+                file_content[
+                    "geophysical_data/Clear_Sky_Confidence"
+                ] = DEFAULT_FILE_DATA
+        elif file_type == "AERDB_":
+            file_content["Latitude"] = DEFAULT_LAT_DATA
+            file_content["Longitude"] = DEFAULT_LON_DATA
+            file_content["Angstrom_Exponent_Land_Ocean_Best_Estimate"] = DEFAULT_FILE_DATA
+            file_content["Aerosol_Optical_Thickness_550_Land_Ocean_Best_Estimate"] = DEFAULT_FILE_DATA
+
+
+class TestVIIRSL2FileHandler:
+    """Test VIIRS_L2 Reader."""
+    yaml_file = "viirs_l2.yaml"
+
+    def setup_method(self):
+        """Wrap NetCDF4 file handler with our own fake handler."""
+        from satpy._config import config_search_paths
+        from satpy.readers.viirs_l2 import VIIRSL2FileHandler
+
+        self.reader_configs = config_search_paths(
+            os.path.join("readers", self.yaml_file)
+        )
+        self.p = mock.patch.object(
+            VIIRSL2FileHandler, "__bases__", (FakeNetCDF4FileHandlerVIIRSL2,)
+        )
+        self.fake_handler = self.p.start()
+        self.p.is_local = True
+
+    def teardown_method(self):
+        """Stop wrapping the NetCDF4 file handler."""
+        self.p.stop()
+
+    @pytest.mark.parametrize(
+        "filename",
+        [
+            ("CLDPROP_L2_VIIRS_SNPP.A2023364.2230.011.2023365115856.nc"),
+            ("CLDMSK_L2_VIIRS_SNPP.A2023364.2230.001.2023365105952.nc"),
+            ("AERDB_L2_VIIRS_SNPP.A2023364.2230.011.2023365113427.nc"),
+        ],
+    )
+    def test_init(self, filename):
+        """Test basic init with no extra parameters."""
+        from satpy.readers import load_reader
+
+        r = load_reader(self.reader_configs)
+        loadables = r.select_files_from_pathnames([filename])
+        assert len(loadables) == 1
+        r.create_filehandlers(loadables)
+        # make sure we have some files
+        assert r.file_handlers
+
+    @pytest.mark.parametrize(
+        ("filename", "datasets"),
+        [
+            pytest.param("CLDPROP_L2_VIIRS_SNPP.A2023364.2230.011.2023365115856.nc",["Cloud_Top_Height"],id="CLDPROP"),
+            pytest.param("CLDMSK_L2_VIIRS_SNPP.A2023364.2230.001.2023365105952.nc",["Clear_Sky_Confidence"],id="CLDMSK"),
+            pytest.param("AERDB_L2_VIIRS_SNPP.A2023364.2230.011.2023365113427.nc",
+                         ["Aerosol_Optical_Thickness_550_Land_Ocean_Best_Estimate",
+                          "Angstrom_Exponent_Land_Ocean_Best_Estimate"],id="AERDB"),
+        ],
+    )
```

Comment:
Our current linter doesn't seem to be complaining about whitespace properly, but these should really have spaces after commas.

```suggestion
            pytest.param("CLDPROP_L2_VIIRS_SNPP.A2023364.2230.011.2023365115856.nc", ["Cloud_Top_Height"], id="CLDPROP"),
            pytest.param("CLDMSK_L2_VIIRS_SNPP.A2023364.2230.001.2023365105952.nc", ["Clear_Sky_Confidence"], id="CLDMSK"),
            pytest.param("AERDB_L2_VIIRS_SNPP.A2023364.2230.011.2023365113427.nc",
                         ["Aerosol_Optical_Thickness_550_Land_Ocean_Best_Estimate",
                          "Angstrom_Exponent_Land_Ocean_Best_Estimate"], id="AERDB"),
        ],
    )
```

Reply:
I updated this, I did notice the linter not picking up on some little things when I ran it in the terminal. I started to just run Black first which seems to pick up on everything and then your linter afterwards to reconcile any difference between the two.